### PR TITLE
Changelog 5.0.0

### PR DESCRIPTION
**Pull request by directionless**
~Drafty McDraft~ Not such a draft any more

### CODE REVIEW

**Comment by theopolis** [comment]
I did another pass, looks good!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,89 @@
 # osquery Changelog
 
+<a name="5.0.2"></a>
+## [5.0.2](https://github.com/osquery/osquery/releases/tag/5.0.2)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.9.0...5.0.2)
+
+Representing commits from 21 contributors! Thank you all.
+
+osquery 5.0 is a tremendously exciting release.
```
Comment (directionless) [inline CHANGELOG.md:10]:
```suggestion
osquery 5.0 is a tremendously exciting release!
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,89 @@
...
+* We now install into /opt/osquery on macOS and Linux for better portability.
+* Our default and recommended installation for macOS uses an application bundle to support entitlement-based features.
+* We now use Endpoint Security APIs for various event-based tables on macOS (more to come in the future!)
+* We now use an osquery-organization macOS code signing certificate.
+
+There are several breaking changes.
+* Installation paths have changes from /usr/local to /opt/osquery on macOS and Linux (symlinks to executables are provided).
```
Comment (directionless) [inline CHANGELOG.md:17]:
```suggestion
* Installation paths have changes from `/usr/local` to `/opt/osquery` on macOS and Linux (symlinks to executables are provided).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,89 @@
...
+* We removed the blacklist key (#7153) in configuration, for several months the use of this key was generating a deprecation warning.
```
Comment (directionless) [inline CHANGELOG.md:18]:
```suggestion
* We removed the deprecated `blacklist` key from the configuration (#7153)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,89 @@
...
+* Search semantics on the augeas table have changed to be more performant, but do break the existing query API.
+
+Representing commits from 20 contributors! Thank you all.
+
+### Table Changes
+
+- Add `secureboot` table for Linux and Windows ([#7202](https://github.com/osquery/osquery/pull/7202))
+- Add `tpm_info` for Windows ([#7107](https://github.com/osquery/osquery/pull/7107))
+- Fix `osquery_info` build_platform column value on Linux ([#7254](https://github.com/osquery/osquery/pull/7254))
+- Support pid_with_namespace in more tables ([#7132](https://github.com/osquery/osquery/pull/7132))
```
Comment (mike-myers-tob) [inline CHANGELOG.md:28]: nit: pid_with_namespace -> `pid_with_namespace`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,89 @@
...
+- Update `augeas` table to use native pattern matching (BREAKING) ([#6982](https://github.com/osquery/osquery/pull/6982))
+- Update `chrome_extensions` to include Edge & EdgeBeta ([#7170](https://github.com/osquery/osquery/pull/7170))
+- Update `disk_encryption` table to support QueryContext ([#7209](https://github.com/osquery/osquery/pull/7209))
+- Update `last` to include utmp type name column ([#7201](https://github.com/osquery/osquery/pull/7201))
+- Update `sudoers` table to support newer include syntax ([#7185](https://github.com/osquery/osquery/pull/7185))
+- Update `user_ssh_keys` to detect encryption of ed25519 keys ([#7168](https://github.com/osquery/osquery/pull/7168))
+
+### Under the Hood improvements
```
Comment (mike-myers-tob) [inline CHANGELOG.md:36]: nit: capitalize "Improvements"

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,89 @@
...
+- Add case sensitive pragma to the pragma/actions authorizer allow list ([#7267](https://github.com/osquery/osquery/pull/7267))
+- Add feature to skip denylist for event-based queries ([#7158](https://github.com/osquery/osquery/pull/7158))
+- Change logger_mode flag to be correctly interpreted as an octal ([#7273](https://github.com/osquery/osquery/pull/7273))
+- Do not let osquery create multiple copies of the extension running at once ([#7178](https://github.com/osquery/osquery/pull/7178))
+- Fix Linux audit rule removal upon osquery exit ([#7221](https://github.com/osquery/osquery/pull/7221))
+- Fix broadcasting empty logs to logger plugins ([#7183](https://github.com/osquery/osquery/pull/7183))
+- Fix issues applying ACLs during chocolatey deployment ([#7166](https://github.com/osquery/osquery/pull/7166))
+- Fix memory issue in Windows fileops ([#7179](https://github.com/osquery/osquery/pull/7179))
+- Fix `process_open_sockets` type error on darwin ([#6546](https://github.com/osquery/osquery/pull/6546))
+- Make sure that the file action "MOVED_TO" is tracked with yara events. ([#7203](https://github.com/osquery/osquery/pull/7203))
```
Comment (mike-myers-tob) [inline CHANGELOG.md:56]: nit: "MOVED_TO" -> `MOVED_TO`

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,89 @@
...
+- Prevent osquery from killing itself when the --force flag is used ([#7295](https://github.com/osquery/osquery/pull/7295))
```
Comment (mike-myers-tob) [inline CHANGELOG.md:57]: --force --> `--force`

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,89 @@
...
+## [5.0.2](https://github.com/osquery/osquery/releases/tag/5.0.2)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.9.0...5.0.2)
+
+Representing commits from 21 contributors! Thank you all.
+
+osquery 5.0 is a tremendously exciting release.
+* We now install into /opt/osquery on macOS and Linux for better portability.
+* Our default and recommended installation for macOS uses an application bundle to support entitlement-based features.
+* We now use Endpoint Security APIs for various event-based tables on macOS (more to come in the future!)
```
Comment (mike-myers-tob) [inline CHANGELOG.md:13]: Specifically we can say:

"We now support process event monitoring using EndpointSecurity APIs on macOS (with more event types to come in the future!)"

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,89 @@
...
+* We now use an osquery-organization macOS code signing certificate.
```
Comment (mike-myers-tob) [inline CHANGELOG.md:14]: Either here or in breaking changes, "Reminder: organizations using MDM to manage osquery should note the impact of the code-signing identity change to their deployments." and a link to the docs https://osquery.readthedocs.io/en/latest/deployment/process-auditing/#automatically-granting-permissions-silent-installs

File: CHANGELOG.md (hunk 4 above)
Comment (directionless) [reply CHANGELOG.md:28]:
```suggestion
- Support `pid_with_namespace` in more tables ([#7132](https://github.com/osquery/osquery/pull/7132))
```

File: CHANGELOG.md (hunk 5 above)
Comment (directionless) [reply CHANGELOG.md:36]:
```suggestion
### Under the Hood Improvements
```

File: CHANGELOG.md (hunk 6 above)
Comment (directionless) [reply CHANGELOG.md:56]:
```suggestion
- Make sure that the file action `MOVED_TO` is tracked with yara events. ([#7203](https://github.com/osquery/osquery/pull/7203))
```

File: CHANGELOG.md (hunk 7 above)
Comment (directionless) [reply CHANGELOG.md:57]:
```suggestion
- Prevent osquery from killing itself when the `--force` flag is used ([#7295](https://github.com/osquery/osquery/pull/7295))
```

**Review by sharvilshah — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,93 @@
 # osquery Changelog
 
+<a name="5.0.1"></a>
+## [5.0.1](https://github.com/osquery/osquery/releases/tag/5.0.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.9.0...5.0.1)
+
+Representing commits from 21 contributors! Thank you all.
```
Comment (Smjert) [inline CHANGELOG.md:8]: This seems repeated below (with a different number)?

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,93 @@
...
+
+There are several breaking changes:
+* Installation paths have changes from `/usr/local` to `/opt/osquery` on macOS and Linux (symlinks to executables are provided).
+* macOS codesigning is now down through the Osquery Foundation account
+* If you manage macOS full disk permission through a profile, you will need to update it.
+  See [docs](https://osquery.readthedocs.io/en/latest/deployment/process-auditing/#automatically-granting-permissions-silent-installs)
+* We removed the deprecated `blacklist` key from the configuration (#7153)
+* Search semantics on the augeas table have changed to be more performant, but do break the existing query API.
+
+Representing commits from 20 contributors! Thank you all.
```
Comment (Smjert) [inline CHANGELOG.md:24]: Repetition? (see above)

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,93 @@
...
+- Update install paths, and remove deprecated Facebook naming ([#7210](https://github.com/osquery/osquery/pull/7210))
+- Update macOS build to include app bundle related files ([#7184](https://github.com/osquery/osquery/pull/7184))
+- Update osquery installed artifacts default paths in code ([#7285](https://github.com/osquery/osquery/pull/7285))
+- Update the installation path on Linux ([#7271](https://github.com/osquery/osquery/pull/7271))
+- libs: Add options to AWS Optionally enable debug option and restrict content-type header size for PUT req ([#7216](https://github.com/osquery/osquery/pull/7216))
+- libs: Enable and compile the YARA macho module on macOS ([#7174](https://github.com/osquery/osquery/pull/7174))
+- libs: Update OpenSSL to version 1.1.1l ([#7293](https://github.com/osquery/osquery/pull/7293))
+- libs: Update Strawberry Perl to 5.32.1.1, use HTTPS downloads ([#7199](https://github.com/osquery/osquery/pull/7199))
+- libs: Update ebpfpub ([#7219](https://github.com/osquery/osquery/pull/7219))
+- libs: Update the ebpfpub library ([#7173](https://github.com/osquery/osquery/pull/7173))
```
Comment (Smjert) [inline CHANGELOG.md:89]: Other nit: maybe collapse?

```suggestion
-libs: Updates to the ebpfpub library ([#7219](https://github.com/osquery/osquery/pull/7219), [#7173](https://github.com/osquery/osquery/pull/7173))
```

**Review by Smjert — DISMISSED** [review]
Only have nits, otherwise looks good!

**Review by Smjert — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (1Husnu) [reply CHANGELOG.md]: https://t.me/c/1309413501/435

**Pull request merged**